monkey/tests/unit_tests/monkey_island/cc/resources/test_agent_events.py

```diff
@@ -33,15 +34,15 @@ class DifferentAgentEvent(AbstractAgentEvent):
     "type": SomeAgentEvent.__name__,
     "some_field": 1,
     "source": "f811ad00-5a68-4437-bd51-7b5cc1768ad5",
-    "target": None,
+    "target": 1,
```

Comment:
There's really no point in changing any of these when we're not verifying that they're being translated in the way we expect them to be. We should just add new tests to test these changes.